cmd/cluster/core/create_test.go

```diff
@@ -155,3 +155,22 @@ func TestAsObjects(t *testing.T) {
 		})
 	}
 }
+
+func TestPrototypeResources(t *testing.T) {
```

Comment:
I've seen you're adding coverage to PrototypeResources function but you're not using it in this PR. Could you separate this part of the PR in a new commit?

Reply:
done!